submarine-server/server-submitter/submitter-k8s/src/main/java/org/apache/submarine/server/submitter/k8s/K8sSubmitter.java

```diff
@@ -361,42 +361,57 @@ public Notebook createNotebook(NotebookSpec spec) throws SubmarineRuntimeExcepti
     final String storage = NotebookUtils.STORAGE;
     final String pvcName = NotebookUtils.PVC_PREFIX + name;
     String namespace = "default";
-    
+
     if (System.getenv(ENV_NAMESPACE) != null) {
       namespace = System.getenv(ENV_NAMESPACE);
     }
-    
+
+    // create notebook custom resource
+    NotebookCR notebookCR = NotebookSpecParser.parseNotebook(spec);
+    Map<String, String> labels = new HashMap<>();
+    labels.put(NotebookCR.NOTEBOOK_OWNER_SELECTOR_KET, spec.getMeta().getOwnerId());
+    notebookCR.getMetadata().setLabels(labels);
+    notebookCR.getMetadata().setNamespace(namespace);
```

Comment:
We should put these codes to try-block to prevent parsing Notebook object failed.